packages/client/src/pages/page-editor/page-editor.container.vue

```diff
@@ -24,48 +24,38 @@
 </div>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
-
-export default defineComponent({
-	props: {
-		expanded: {
-			type: Boolean,
-			default: true
-		},
-		removable: {
-			type: Boolean,
-			default: true
-		},
-		draggable: {
-			type: Boolean,
-			default: false
-		},
-		error: {
-			required: false,
-			default: null
-		},
-		warn: {
-			required: false,
-			default: null
-		}
-	},
-	emits: ['toggle', 'remove'],
-	data() {
-		return {
-			showBody: this.expanded,
-		};
-	},
-	methods: {
-		toggleContent(show: boolean) {
-			this.showBody = show;
-			this.$emit('toggle', show);
-		},
-		remove() {
-			this.$emit('remove');
-		}
-	}
+<script lang="ts" setup>
+import { } from 'vue';
+
+const props = withDefaults(defineProps<{
+	expanded: boolean,
+	removable: boolean,
+	draggable: boolean,
+	error: any,
+	warn: any
```

Comment:
> All props are optional by default, unless required: true is specified.
https://vuejs.org/guide/components/props.html#prop-validation

Some of the properties are even explicitly defined with `required: false`.
```suggestion
	expanded?: boolean,
	removable?: boolean,
	draggable?: boolean,
	error?: any,
	warn?: any
```